homework-submission/week1/todoapp/src/components/AddTodo.js

```diff
@@ -0,0 +1,45 @@
+/*This file get todo from user input and add it to list,
+ *!!!> commands are not important! just for self learning */
+
+// we need a local state for this component to store what are user types into the import field and since we need state this should be a class-based component
+// extends component>> to inherit the functionality of a component in react
+//render()>> JSX template
+import React, { Component } from 'react';
+class AddTodo extends Component {
```

Comment:
Cool that you already implemented a form to add a todo! Very well implemented as well!